Makefile

```diff
@@ -4,7 +4,7 @@ $(MODULES):
 	$(MAKE) -C $@ $(MAKECMDGOALS)
 
 fmt-root:
-	cljfmt check dev resources src test deps.edn tests.edn
+	cljfmt check dev modules profiling resources src test deps.edn tests.edn
```

Comment:
No need the check the whole modules dir here, because that is done at module level. `profiling` is ok.

Reply:
And I miss the fixed profiling namespace.

Reply:
See TODO here:
https://github.com/samply/blaze/pull/2125#discussion_r1806311303